### PR TITLE
test: add vpc sweepers

### PR DESCRIPTION
**Pull request by ivan-savciuc**
## About this change—what it does

Add sweepers for VPC package resources

### CODE REVIEW

File: internal/sdkprovider/service/vpc/sweep.go [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+func sweepVPCs(ctx context.Context, client *aiven.Client) func(string) error {
+	return func(region string) error {
+		projectName := os.Getenv("AIVEN_PROJECT_NAME")
+		vpcs, err := client.VPCs.List(ctx, projectName)
+		if err != nil {
+			return fmt.Errorf("error retrieving a list of vpcs for a project : %s", err)
+		}
+
+		for _, vpc := range vpcs {
+
```
Comment (Serpentiel) [inline internal/sdkprovider/service/vpc/sweep.go:110]: remove new line

File: internal/sdkprovider/service/vpc/sweep.go [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+			err := client.VPCs.Delete(ctx, projectName, vpc.ProjectVPCID)
+			if err != nil && !aiven.IsNotFound(err) {
```
Comment (Serpentiel) [inline internal/sdkprovider/service/vpc/sweep.go:112]: let's do this: https://github.com/aiven/terraform-provider-aiven/pull/1525#discussion_r1448996423

**Review by Serpentiel — CHANGES_REQUESTED** [review]
some minor changes

**Comment by ivan-savciuc** [comment]
@Serpentiel waiting for this merger first https://github.com/aiven/terraform-provider-aiven/pull/1525 to avoid conflicts

**Comment by Serpentiel** [comment]
@ivan-savciuc done

**Comment by Serpentiel** [comment]
duplicates #1529

**Pull request closed without merging**